src/pcrlock/pcrlock.c

```diff
@@ -1324,6 +1349,10 @@ static int event_log_calculate_pcrs(EventLog *el) {
                 }
 
         FOREACH_ARRAY(rr, el->records, el->n_records) {
+
+                if (!EVENT_LOG_RECORD_IS_PCR(*rr))
```

Comment:
EVENT_LOG_RECORD_IS_PCR() and EVENT_LOG_RECORD_IS_NV_INDEX() are not defined in this commit.